htdocs/manage/emailpost.bml

```diff
@@ -367,6 +369,28 @@ body<=
                     : "<div class='error-box'>$ML{'.reply.status.error'}</div>";
     }
 
+    if ( LJ::did_post() && $POST{"action:apikey"}) {
+        $ret .= DW::API::Key->new_for_user($u)
```

Comment:
need to check form auth

Reply:
🆗

Reply:
Need to abort the flow if form auth is invalid so we don't do the action anyway.